src/views/signup/SurveyView1.vue

```diff
@@ -1,18 +1,178 @@
 <template>
-  <div class="flex justify-center">
-    <div class="bg-ivory flex flex-col gap-7 w-full px-4">
+  <div class="flex justify-center w-full">
+    <div class="bg-ivory flex flex-col gap-6 w-full px-6">
       <!-- 타이틀 -->
-      <div class="gap-9">
-        <div class="flex font-bold text-2xl justify-center">사전 조사</div>
-
+      <div class="flex flex-col text-center gap-2">
+        <div class="tfont-bold text-2xl justify-center">사전 조사</div>
         <div class="text-limegreen-700 text-sm">
           현재 자산 유형 분석을 위한 질문입니다.
         </div>
       </div>
-      <!--설문조사 문항-->
-      <div></div>
+
+      <!-- 설문조사 문항 -->
+      <div class="flex flex-col gap-5">
+        <div
+          v-for="question in questions"
+          :key="question.id"
+          class="flex flex-col gap-3 bg-ivory rounded-lg"
+        >
+          <!-- 질문 제목 -->
+          <div
+            class="px-3 py-1 rounded-lg flex bg-limegreen-100 justify-between items-center"
+          >
+            <div class="flex items-center">
+              <div class="font-thin text-sm text-limegreen-800">
+                {{ question.title }}
+              </div>
+              <div
+                v-if="question.subtitle"
+                class="text-xs text-gray-300 font-thin px-1"
+              >
+                {{ question.subtitle }}
+              </div>
+            </div>
+            <div v-if="question.priceUnit" class="text-xs text-gray-300">
+              {{ question.priceUnit }}
+            </div>
+          </div>
+
+          <div class="relative">
+            <div
+              class="absolute top-3 left-6 right-6 h-0.5 bg-limegreen-500 z-0"
+            ></div>
+
+            <!-- 설문조사 응답 버튼과 라벨 -->
+            <div class="grid grid-cols-5 gap-0 relative z-10">
+              <div v-for="(option, index) in question.options" :key="index">
+                <!-- 각 옵션을 컨테이너로 묶기 -->
+                <div class="flex flex-col items-center">
+                  <div
+                    @click="selectOption(question.id, option.value)"
+                    :class="[
+                      'w-6 h-6 rounded-full border-2 text-gray-300 border-limegreen-500 flex items-center justify-center text-xs cursor-pointer ',
+                      answers[question.id] === option.value
+                        ? 'bg-limegreen-100'
+                        : 'bg-white hover:bg-limegreen-100',
+                    ]"
+                  >
+                    {{ index + 1 }}
```

Comment:
이 부분은 index에 + 1보다는 데이터의 value 값을 보여주는 것이 더 잘 맞을 거 같아요!
```suggestion
                    {{ option.value }}
```